modules/apps/headless/headless-builder/headless-builder-web/src/main/resources/META-INF/resources/js/components/baseComponents/BaseAPIEndpointFields.tsx

```diff
@@ -25,6 +25,7 @@ interface BaseAPIApplicationFieldsProps {
 	apiApplicationBaseURL: string;
 	basePath: string;
 	data: Partial<APIEndpointUIData>;
+	disabledProperties: Array<string>;
```

Comment:
As we are working with the boolean value, I would change this to `isEdition?: Boolean;`
The `?` makes it possible to no setting the property as we are doing in `CreateAPIEndpoint`

Reply:
Nice!! I will rename it as `editing` as there are some records in liferay-portal using this semantic convention